src/app/api/uploadfile/post.controller.js

```diff
@@ -6,23 +6,34 @@ const garApi = require('../../../common/services/garApi');
 const transformers = require('../../../common/utils/transformers');
 const clamAVService = require('../../../common/services/clamAVService');
 const uploadFile = require('../../../common/services/fileUploadApi');
+const config = require('../../../common/config/index');
 const { isValidFileMime } = require('../../../common/utils/validator');
 
-const exceedFileLimit = (fileSize, garId) => {
-  logger.debug('Entering exceed file limit function');
+const exceedFileNumSizeLimit = (fileSize, garId) => {
+  logger.debug('Entering exceed file number & size limit function');
```

Comment:
It might be good to have an indication of what the max alllowed value is in this log entry?

Reply:
Done